frontend/components/RewardsHistory/types.ts

```diff
@@ -0,0 +1,20 @@
+import { z } from 'zod';
```

Comment:
what's the benefit of using this? https://zod.dev/?id=introduction it says `Works with plain JavaScript too! You don't need to use TypeScript.` but we have typescript 🤔